src/App.js

```diff
@@ -14,8 +14,9 @@ import CreateAdmin from './AdminPage/CreateAdmin';
 import UpdateEvent from './UpdateEventPage/UpdateEvent';
 import EditPage from './EditPage/EditPage';
 import DeletedUser from './ProfilesPage/DeletedUser';
-import ChatWidget from './ChatWidget/ChatWidget';
+// import ChatWidget from './ChatWidget/ChatWidget';
```

Comment:
Pulling from `qa` will probably pop up some merge conflicts here; just go ahead and delete these commented-out lines as well.